change/@ni-nimble-components-225009b9-77d6-4cbd-a94e-d9ba2751b047.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "Initial submission of accordion ixd spec",
+  "packageName": "@ni/nimble-components",
+  "email": "leslie.bloomfield@ni.com",
```

Comment:
Most of us have chosen to configure our system to use an obfuscated github email rather than expose our NI emails to the world.

To fix this [go to github settings](https://github.com/settings/emails) and click "Keep my email address private". Then you can copy the obfuscated email from there and manually update it here. There are also instructions there on how to configure git to always use the obfuscated email in the future.